CONTRIBUTING.md

```diff
@@ -1,150 +1,121 @@
-# Contribution Guidelines
+# 🤝 Contributing to דאטאבוס
 
-Hello :wave: and welcome to **דאטאבוס**, a project of [hasadna](https://open-bus-map-search.hasadna.org.il)!
+Welcome 👋 and thank you for considering contributing to **דאטאבוס**, a project by [Hasadna](https://open-bus-map-search.hasadna.org.il)!  
+We’re excited to have you on board.
 
-We glad to see you decided to contribute to our project!
+💬 Join our [Discord](https://discord.gg/deBdkmufS4) to chat with the dev team.
```

Comment:
I think we should remove this discord link and encourage people to join our slack channel instead